abci_test.go

```diff
@@ -36,7 +36,7 @@ func TestCreateErrorResult(t *testing.T) {
 			assert.Contains(t, dres.Log, "iov-one/weave/abci")
 			assert.Equal(t, tc.code, dres.Code)
 
-			cres := weave.CheckTxError(tc.err)
+			cres := weave.CheckTxError(tc.err, true)
```

Comment:
Good to extend this for the true and false case, that they both start with the same string (expected from the error), and the stack trace is only present with debug=true